scripts/sfpshow

```diff
@@ -408,24 +409,22 @@ class SFPShow(object):
         if self.intf_name is not None:
             presence = self.db.exists(self.db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(self.intf_name))
             if presence:
-                self.output = self.convert_interface_sfp_info_to_cli_output_string(
+                self.intf_eeprom[self.intf_name] = self.convert_interface_sfp_info_to_cli_output_string(
                     self.db, self.intf_name, self.dump_dom)
             else:
-                self.output += (self.intf_name + ': ' + 'SFP EEPROM Not detected' + '\n')
+                self.intf_eeprom[self.intf_name] = "SFP EEPROM Not detected\n"
         else:
             port_table_keys = self.db.keys(self.db.APPL_DB, "PORT_TABLE:*")
-            sorted_table_keys = natsorted(port_table_keys)
-            for i in sorted_table_keys:
+            for i in port_table_keys:
                 interface = re.split(':', i, maxsplit=1)[-1].strip()
                 if interface and interface.startswith(front_panel_prefix()) and not interface.startswith((backplane_prefix(), inband_prefix(), recirc_prefix())):
                     presence = self.db.exists(self.db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(interface))
                     if presence:
-                        self.output += self.convert_interface_sfp_info_to_cli_output_string(
+                        self.intf_eeprom[interface] = self.convert_interface_sfp_info_to_cli_output_string(
                             self.db, interface, self.dump_dom)
                     else:
-                        self.output += (interface + ': ' + 'SFP EEPROM Not detected' + '\n')
+                        self.intf_eeprom[interface] = "SFP EEPROM Not detected\n"
 
-                    self.output += '\n'
 
     @multi_asic_util.run_on_multi_asic
     def get_presence(self):
```

Comment:
can you fix get_presence(). I think that is also NOT sorted in your platform case

Reply:
@nathcohe fixing in this pr?